acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/SignerConfiguration.java

```diff
@@ -12,15 +12,16 @@
  */
 package tech.pegasys.ethsigner.tests.dsl.signer;
 
-import tech.pegasys.ethsigner.core.TlsOptions;
+import tech.pegasys.ethsigner.core.config.PkcsStoreConfig;
+import tech.pegasys.ethsigner.core.config.TlsOptions;
 
 import java.time.Duration;
 import java.util.Optional;
 
 public class SignerConfiguration {
 
   private static final Duration POLLING_INTERVAL = Duration.ofMillis(500);
-  private static final Duration TIMEOUT = Duration.ofSeconds(2);
+  private static final Duration TIMEOUT = Duration.ofSeconds(5);
```

Comment:
I am usually a fan of using system properties to override such properties, for now this is alright.

Reply:
oooh - that shouldn't have changed ...